src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1011,48 +1012,57 @@ bool RaftPart::prepareElectionRequest(
     req.set_part(partId_);
     req.set_candidate_ip(addr_.first);
     req.set_candidate_port(addr_.second);
-    req.set_term(term_ + 1);
+    if (isPreVote) {
+        req.set_term(term_ + 1);
+    } else {
+        req.set_term(++term_);
```

Comment:
Bump up term_ before commit step?  What if timeout in this step?  The term_ will be increased crazily, it is same with the original purposeTerm.

Reply:
Make sense, so use the term in prepare phase?